x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/ShardChangesIT.java

```diff
@@ -490,7 +491,7 @@ public void testFollowIndex_lowMaxTranslogBytes() throws Exception {
                 ShardFollowNodeTask.Status status = (ShardFollowNodeTask.Status) taskInfo.getStatus();
                 assertThat(status, notNullValue());
                 assertThat("incorrect global checkpoint " + shardFollowTaskParams,
-                    status.getFollowerGlobalCheckpoint(),
+                    status.followerGlobalCheckpoint(),
```

Comment:
just wondering (I don't care much) - why did you move from the get* method naming?